src/sql/workbench/common/editor/query/queryEditorInput.ts

```diff
@@ -264,8 +269,36 @@ export abstract class QueryEditorInput extends EditorInput implements IConnectab
 	}
 
 	// Called to get the tooltip of the tab
-	public override getTitle(): string {
-		return this.getName(true);
+	public override getTitle(verbosity?: Verbosity): string {
+		let profile = this.connectionManagementService.getConnectionProfile(this.uri);
+		let fullTitle = '';
+		if (profile) {
+			let additionalOptions = this.connectionManagementService.getNonDefaultOptions(profile);
+			if (this._description && this._description !== '') {
+				fullTitle = this._description + ' ';
+			}
+			fullTitle += `${profile.serverName}`;
+			if (profile.databaseName) {
+				fullTitle += `.${profile.databaseName}`;
```

Comment:
Is the `.` needed for formatting or something that was added accidentally?

Reply:
This is needed for formatting, it is to match the title found in getName()